examples/psyir/README.md

```diff
@@ -63,3 +63,14 @@ To run this example:
 ```sh
 > python create_aggregate_types.py
 ```
+
+## Example 3:
```

Comment:
Update copyright date and authorship please.

Reply:
Done